docker/amazonlinux-latest-amd64-openssl-1.0.x/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# syntax=docker/dockerfile:1
+
+# substitute for centos
+FROM amazonlinux:latest
+
+# Node.js 18 isn't compatible with Amazon Linux due to glibc version. It would throw with:
+#   node: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by node)
+#   node: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by node)
```

Comment:
Oh I didn't know about this! 
So Node.js v18 requires GLIBC_2.27 at minimum but Amazon Linux 2 comes with 2.26

Reply:
So, after reading [this](https://aws.amazon.com/about-aws/whats-new/2022/11/aws-lambda-support-node-js-18/) (AWS Lambda adds support for Node.js 18), I've figured that Amazon's Docker image naming is more inconsistent than anticipated.

- The Docker image for Amazon Linux 2 is `amazonlinux` (`:latest`, `:2`)
- The Docker image for Amazon Linux 2022 is `public.ecr.aws/amazonlinux/amazonlinux:2022`

From the [FAQs](https://aws.amazon.com/linux/amazon-linux-2022/faqs/):

> Amazon Linux 2022 (AL2022) is a general-purpose rpm-based Linux distribution and a successor to Amazon Linux 2. AL2022 simplifies planning for operating system upgrades. Starting with AL2022, a new major version of Amazon Linux is released every two years, includes minor quarterly releases, and comes with five years of long-term support. Amazon Linux 2022 integrates with AWS services and is designed to be deployed at scale in the cloud.

Reply:
I will add both images to this PR.

Reply:
Note: this is the same information posted by @Jolg42 in [this comment](https://github.com/prisma/ecosystem-tests/pull/3402#issuecomment-1400686891) a few minutes later.